assemble/bin/accumulo-cluster

```diff
@@ -100,15 +100,45 @@ function get_ip() {
   echo "$ip_addr"
 }
 
+function set_last_instance_id() {
+  if [[ "$service" == "tserver" ]]; then
+    last_instance_id=${NUM_TSERVERS:-1}
+  else
+    last_instance_id=1
+  fi
+}
+
+function set_service_instance_if_needed() {
+  if [[ "$service" == "tserver" && ${NUM_TSERVERS:-1} > 1 ]]; then
+    export ACCUMULO_SERVICE_INSTANCE=$inst_id
+    service_instance_cmd="export ACCUMULO_SERVICE_INSTANCE=${inst_id};"
+  else
+    export ACCUMULO_SERVICE_INSTANCE=
+    service_instance_cmd=
```

Comment:
Not sure the following will work, the thought is to make running on localhost and remote consistent.  Both ensuring the env var is not set when running the script.

```suggestion
    service_instance_cmd="export ACCUMULO_SERVICE_INSTANCE=;"
```

Reply:
Will consider and implement. Thanks!

Reply:
Thanks Keith. With the suggestions that Christopher made, this has been handled in a different way now. Please take a look.

Reply:
Those changes look good and consistent.